src/argilla/client/feedback/dataset/base.py

```diff
@@ -376,11 +393,15 @@ def prepare_for_training(
                     lang = spacy.blank(lang)
                 else:
                     lang = spacy.load(lang)
-            return task_mapping._prepare_for_training_with_spacy(data=data, train_size=train_size, seed=seed, lang=lang)
+            return task._prepare_for_training_with_spacy(data=data, train_size=train_size, seed=seed, lang=lang)
         elif framework is Framework.SPARK_NLP:
-            return task_mapping._prepare_for_training_with_spark_nlp(data=data, train_size=train_size, seed=seed)
+            return task._prepare_for_training_with_spark_nlp(data=data, train_size=train_size, seed=seed)
```

Comment:
IMO we could safely drop `spaCy` and `SparkNLP` support, right? If we want to keep those for a strong reason we can discuss about it

Reply:
I think it should be fine to remove. The spaCy transformer tests actually cause my pytest to abruptly terminate. Not ideal.